int32 QUERY=0          # State constant - query on actual state
int32 INITPREPARE=1    # State constant - initial preparation
int32 SEARCHBALL=2     # State constant - searching for balls
int32 TURNBALL=3       # State constant - turn robot around
int32 MOVETOBALL=4     # State constant - move towards ball
int32 CHECKBALL=5      # State constant - check position of the ball
int32 PICKBALL=6       # State constant - pick the ball
int32 SEARCHHAND=7     # State constant - search for hand
int32 TURNHAND=8       # State constant - turn robot around
int32 MOVETOHAND=9     # State constant - move the ball towards the hand
int32 CHECKHAND=10     # State constant - check the position of the ball
int32 PLACEBALL=11     # State constant - place ball into the hand
int32 DROPBALL=12      # State constant - drop the ball
int32 flowid
